build/tutorials/tutorials-contest/pangolin-token-subgraph/README.md

```diff
@@ -0,0 +1,480 @@
+
+In GraphQL, the Query type is the entry point to the API, The Graph defines an Entity type for its schema but automatically creates the Query type as the top-level type (root type) for you with fields - entity and entities. Below is an example.
+
+```graphql
+type Token @entity {
+  id: ID!
+  address: Bytes!
+  name: String!
+  symbol: String
+}
+```
+
+The entity in this example is Token, the @entity directive is provided by The Graph and will create fields token and tokens in the automatically generated Query type. Note that these fields generated are what will be used in constructing a GraphQL query as you will see shortly. The fields in the Token entity - id, address, name, symbol are the data that will be returned if they are included as part of a query. An entity must also include an id field for it to be considered valid. The exclamation mark indicates that the field is NON NULLABLE, which means it will always have a value.
+
+By looking at the GraphQL schema above, you can construct a query that closely mirrors it and includes only the data you are interested in. An example query is shown below.
+
+```graphql
+{
+  token(id: "1") {
+    id
+    address
+  }
+}
+```
+
+The example query is for a specific token entity with an `id` of `1`. When querying for a single entity in The Graph, the `id` field must be included. The result returned will mirror the query structure and will only include data requested for which in this case is the `id` and `address` fields. A  probable result is shown below.
+
+```graphql
+{
+  "data": {
+    "token": {
+    "id": "0x10",
+    "address": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2"
+   }
+  }
+}
+```
+
+You can also query for all tokens (entities) using the query below.
+
+```graphql
+{
+  tokens {
+    id
+    address
+    name
+    symbol
+  }
+}
+```
+
+The results will match the query structure and will be an array of tokens with the additional fields - `name` and `symbol` included because they were requested for unlike in the last query example.
+
+```graphql
+{
+  "data": {
+    "tokens": [
+      {
+        "id": "0x10",
+        "address": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+        "name": "Wrapped Ether",
+        "symbol": "WETH"
+      },
+      {
+        "id": "0x11",
+        "address": "0x2260fac5e5542a773aa44fbcfedf7c193bc2c599",
+        "name": "Wrapped BTC",
+        "symbol": "WBTC"
+      }
+    ]
+  }
+}
+```
+
+The Graph also includes support for sorting, pagination, filtering, time-travel queries, etc. please refer to [The Graph documentation](https://thegraph.com/docs/developer/graphql-api) for a full list of features.
+
+### What is Avalanche
```

Comment:
This section can be much shorter. Just link out to [our existing documentation](https://docs.avax.network/learn/platform-overview)

Reply:
Made the section shorter and pointed to the official documentation.